
时间：2017-05-23 10:46:36,160 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：29018 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.DBDataContext.GetDictList(String code, String upval) 位置 D:\Pro\x.link\src\Web\X.App\Com\DB.cs:行号 98
   在 X.App.Apis.cate.list.tree_LoadList(Object id) 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\list.cs:行号 35
   在 X.App.XTree.getChild(TreeNode n, Int32 d) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 63
   在 X.App.XTree.InitTree(String title, Int32 deep) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 27
   在 X.App.Apis.cate.list.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\list.cs:行号 24
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20

时间：2017-05-23 10:48:41,571 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：694 
System.ArgumentNullException: 值不能为 null。
参数名: source
   在 System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   在 X.App.Apis.cate.list.tree_LoadList(Object id) 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\list.cs:行号 35
   在 X.App.XTree.getChild(TreeNode n, Int32 d) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 63
   在 X.App.XTree.InitTree(String title, Int32 deep) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 27
   在 X.App.Apis.cate.list.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\list.cs:行号 24
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 10:52:24,318 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：153578 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.Web.Com.XForm.pick.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 301
   在 X.Web.Com.XForm.<>c.<loadcts>b__6_0(Match o) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 58
   在 System.Text.RegularExpressions.RegexReplacement.Replace(MatchEvaluator evaluator, Regex regex, String input, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator)
   在 X.Web.Com.XForm.loadcts(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 55
   在 X.Web.Com.XForm.form.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 135
   在 X.Web.Com.XForm.Parse(String html) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 79
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 119
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 10:52:38,779 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：2431 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.Web.Com.XForm.pick.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 301
   在 X.Web.Com.XForm.<>c.<loadcts>b__6_0(Match o) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 58
   在 System.Text.RegularExpressions.RegexReplacement.Replace(MatchEvaluator evaluator, Regex regex, String input, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator)
   在 X.Web.Com.XForm.loadcts(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 55
   在 X.Web.Com.XForm.form.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 135
   在 X.Web.Com.XForm.Parse(String html) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 79
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 119
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 10:52:48,196 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：11848 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.Web.Com.XForm.pick.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 301
   在 X.Web.Com.XForm.<>c.<loadcts>b__6_0(Match o) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 58
   在 System.Text.RegularExpressions.RegexReplacement.Replace(MatchEvaluator evaluator, Regex regex, String input, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator)
   在 X.Web.Com.XForm.loadcts(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 55
   在 X.Web.Com.XForm.form.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 135
   在 X.Web.Com.XForm.Parse(String html) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 79
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 119
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 10:53:35,610 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：2448 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.Web.Com.XForm.pick.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 301
   在 X.Web.Com.XForm.<>c.<loadcts>b__6_0(Match o) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 58
   在 System.Text.RegularExpressions.RegexReplacement.Replace(MatchEvaluator evaluator, Regex regex, String input, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator, Int32 count, Int32 startat)
   在 System.Text.RegularExpressions.Regex.Replace(String input, MatchEvaluator evaluator)
   在 X.Web.Com.XForm.loadcts(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 55
   在 X.Web.Com.XForm.form.init(String body) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 135
   在 X.Web.Com.XForm.Parse(String html) 位置 D:\Pro\x.link\src\X.Web\Com\XForm.cs:行号 79
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 119
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 11:35:57,994 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：65370 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Apis.cate.save.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 11:53:01,279 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：19535 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Apis.cate.save.<>c__DisplayClass20_0.<Execute>b__0(x_dict o) 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\save.cs:行号 45
   在 System.Linq.Enumerable.WhereListIterator`1.MoveNext()
   在 System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   在 System.Linq.OrderedEnumerable`1.<GetEnumerator>d__1.MoveNext()
   在 X.App.Apis.cate.save.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\save.cs:行号 47
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 12:15:15,366 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：340927 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Apis.cate.save.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 12:17:37,538 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：33928 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Apis.cate.save.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 12:28:15,177 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：407352 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Apis.cate.save.Execute()
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 14:41:31,693 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：69430 
System.Data.SqlClient.SqlException (0x80131904): 将截断字符串或二进制数据。
语句已终止。
   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   在 System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   在 System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.DynamicUpdate(TrackedObject item)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.Update(TrackedObject item)
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 X.App.DBDataContext.SubmitDBChanges() 位置 D:\Pro\x.link\src\Web\X.App\Com\DB.cs:行号 110
   在 X.App.Apis.cate.save.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\save.cs:行号 73
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31
ClientConnectionId:63cab09d-8db6-4070-ace4-1ab7fc76dd8c
Error Number:8152,State:13,Class:16

时间：2017-05-23 14:41:57,354 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：95091 
System.Data.SqlClient.SqlException (0x80131904): 将截断字符串或二进制数据。
语句已终止。
   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   在 System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   在 System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.DynamicUpdate(TrackedObject item)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.Update(TrackedObject item)
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 X.App.DBDataContext.SubmitDBChanges() 位置 D:\Pro\x.link\src\Web\X.App\Com\DB.cs:行号 110
   在 X.App.Apis.cate.save.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\cate\save.cs:行号 73
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31
ClientConnectionId:63cab09d-8db6-4070-ace4-1ab7fc76dd8c
Error Number:8152,State:13,Class:16

时间：2017-05-23 16:39:54,419 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(:0) - 
错误：值不能为 null。
参数名: source
   在 System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   在 X.App.Views.com.dict.tree_LoadList(Object id) 位置 D:\Pro\x.link\src\Web\X.App\Views\com\dict.cs:行号 52
   在 X.App.XTree.getChild(TreeNode n, Int32 d) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 63
   在 X.App.XTree.InitTree(String title, Int32 deep) 位置 D:\Pro\x.link\src\Web\X.App\Com\XTree.cs:行号 27
   在 X.App.Views.com.dict.InitDict() 位置 D:\Pro\x.link\src\Web\X.App\Views\com\dict.cs:行号 40
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 102 
时间：6825 

时间：2017-05-23 16:39:54,468 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：6875 
System.ArgumentNullException: 值不能为 null。
参数名: source
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Views\View.cs:行号 112
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31

时间：2017-05-23 16:41:12,467 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(:0) - 
错误：{"issucc":false,"msg":"接口不存在 url.save","code":"0x0002"} 
时间：84873 

时间：2017-05-23 16:41:53,966 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：258 
System.Data.SqlClient.SqlException (0x80131904): INSERT 语句与 FOREIGN KEY 约束"FK_x_url_x_user"冲突。该冲突发生于数据库"x.link"，表"dbo.x_user", column 'user_id'。
语句已终止。
   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   在 System.Data.SqlClient.SqlDataReader.get_MetaData()
   在 System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   在 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   在 System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   在 System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   在 System.Data.Common.DbCommand.ExecuteReader()
   在 System.Data.Linq.SqlClient.SqlProvider.Execute(Expression query, QueryInfo queryInfo, IObjectReaderFactory factory, Object[] parentArgs, Object[] userArgs, ICompiledSubQuery[] subQueries, Object lastResult)
   在 System.Data.Linq.SqlClient.SqlProvider.ExecuteAll(Expression query, QueryInfo[] queryInfos, IObjectReaderFactory factory, Object[] userArguments, ICompiledSubQuery[] subQueries)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.DynamicInsert(TrackedObject item)
   在 System.Data.Linq.ChangeDirector.StandardChangeDirector.Insert(TrackedObject item)
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 X.App.DBDataContext.SubmitDBChanges() 位置 D:\Pro\x.link\src\Web\X.App\Com\DB.cs:行号 110
   在 X.App.Apis.url.save.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\url\save.cs:行号 38
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31
ClientConnectionId:7bcda0c8-f2ef-4337-b476-0a2352bc8dba
Error Number:547,State:0,Class:16

时间：2017-05-23 16:47:07,811 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(:0) - 
错误：{"issucc":false,"msg":"接口不存在 poi.list","code":"0x0002"} 
时间：48195 

时间：2017-05-23 16:54:13,226 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(:0) - 
错误：{"issucc":false,"msg":"未知错误代码(T网址不存在)","code":"T网址不存在"} 
时间：167429 

时间：2017-05-23 17:28:08,955 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(:0) - 
错误：{"issucc":false,"msg":"接口不存在 tag.save","code":"0x0002"} 
时间：50957 

时间：2017-05-23 17:30:36,521 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(:0) - 
错误：致命错误 
时间：70454 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 X.App.DBDataContext.SubmitDBChanges() 位置 D:\Pro\x.link\src\Web\X.App\Com\DB.cs:行号 110
   在 X.App.Apis.tag.save.Execute() 位置 D:\Pro\x.link\src\Web\X.App\Apis\tag\save.cs:行号 35
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.link\src\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.link\src\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
